content/posts/irods-development-update-november-2021.md

```diff
@@ -0,0 +1,279 @@
+Title: iRODS Development Update: November 2021
+Date: 2021-11-24 12:00
+Author: Kory Draughn
+Slug: irods-development-update-november-2021
+Status: published
+
+
+So Supercomputing 2021 happened and while it was very different, it was great being able to see so many people in tech again.
+
+This month we saw many improvements to the main server. Server-to-server negotiation and semantics around managing users and groups have been tightened up. The server also grew a new configuration option that allows administrators to apply a limit on the amount of memory available to the delay server. No more out-of-memory errors! The checksum API received some updates to make reacting to verification results better as well.
+
+The Docker-based Testing Environment has grown support for SSL. We've modularized the code a bit more so that it is easier to maintain and development is still progressing smoothly. If you can run docker, then you can run the test suite. If that sounds interesting, give it a try. Your feedback will help us produce a better tool.
+
+The Indexing Capability Plugin has received numerous fixes, major and minor. One exciting aspect is that it is now interoperable with Metalnx and the NIEHS developed global-search endpoint. Expect a release of Indexing Capability 4.2.10.1 very soon.
+
+The C++ REST API and Python iRODS Client have grown full support for tickets. We are actively preparing releases for both projects.
+
+We've finally seen NFSRODS successfully transfer a file in parallel. This is all possible due to Jargon's newly added support for replica access tokens and multi-1247 parallel I/O. We have a few more things to work out for NFSRODS, so stay on the lookout for the next release. It's going to be so good!
+
+Last but not least, there's been some discussion around replacing imcoll and the structured file APIs. Take a look at the Discussion section for information around that.
+
+Well, that wraps up this month's update. Have some turkey and we'll see you next month!
```

Comment:
not sure about the "Have some turkey" part.

@trel what do you think?

Reply:
agreed - just a US holiday.   maybe something about how we're running fast to get things out by the end of the year.

rest looks good.

Reply:
awesome. i will adjust that last statement.

Reply:
updated.